tests/README.md

```diff
@@ -2,9 +2,25 @@
 - baseclasses
 - testflo
 
-# Instructions
-- To train regression tests: `testflo reg_tests -m train`
-- To run regression tests: `testflo reg_tests`
-- To run unit tests: `testflo unit_tests`
-- To run regression and unit tests: `testflo reg_tests unit_tests`
-- To run individual test: `testflo reg_tests/test1.py`
\ No newline at end of file
+
+# Instructions and examples
+- To train all regression tests with specific function signature: `testflo reg_tests -m "train_*"`
+- To train all regression tests in specific testfile either works: `cd reg_tests; testflo test_DVGeometry.py -m "train_*"` or `testflo reg_tests/test_DVGeometry.py -m "train_*"` 
+- To train specific regression test in specific testfile either works: `cd reg_tests; testflo test_DVGeometry.py -m "test_1` or `testflo reg_tests/test_DVGeometry.py -m "train_1"`
+
+- To test all regression tests: `testflo reg_tests`
+- To test all regression tests with specific function signature: `testflo reg_tests -m "test_*"`
+- To test all regression tests in specific testfile either works: `cd reg_tests; testflo test_DVGeometry.py -m "test_*"` or `testflo reg_tests/test_DVGeometry.py -m "test_*"` 
+- To test specific regression test in a specific testfile either works: `cd reg_tests; testflo test_DVGeometry.py -m "test_1` or `testflo reg_tests/test_DVGeometry.py -m "test_1"`
+
+- To enable screen output: use `-s` argument
+- To change number of cores to 1 for running tests: use `-n 1`
```

Comment:
It would be useful to provide a single command which would run all ADflow tests, and return the result to screen.

Reply:
you can run all the regression tests with `testflo reg_tests -s`.
Is that what you mean?

Reply:
I haven't tried that to see what it would output, but basically a command that replaces the `python run_reg_tests.py` command we had before, such that ADflow users can verify their installation. We can have additional commands for developers and others to debug things, but it would be useful to recommend a single command to use just to verify everything.

Reply:
I think the best option would be to just make a new `run_reg_tests.py` even if it just makes a call to the shell.That would make it clear for those just looking to run all the tests.

People who want more options can use the README.md to figure out what to pass to testflo.